src/app/assets/languages/en.json

```diff
@@ -268,6 +268,12 @@
       "modal-enable-button" : "Enable",
       "modal-cancel-button" : "Cancel",
       "modal-description" : " Disabling rebalancing prevents the cluster from repairing data in this cache belonging to nodes leaving the cluster. It should be re-enabled as soon as possible to prevent possible data-loss in this cache."
+    },
+    "cache-metrics":{
+      "current-number-entries":"Current number of entries",
+      "min-nodes":"Minimum number of nodes",
+      "off-heap-memory":"Amount of off-heap memory used",
+      "data-memory":"Amount of heap memory used"
     }
```

Comment:
Sorry again @dipguptaredhat But I think we could be more meaningful here yet so a few more nitpicks and small changes. I can't seem to add inline suggestions so I'll just put everything in a single comment. Obvs there are changes here that affect the tsx files too.

How about...

      "off-heap-memory":"Amount of off-heap memory used", -> "cache-size-off-heap":"Size of cache in off-heap memory"
      "data-memory":"Amount of heap memory used" -> "cache-size-heap":"Size of cache in heap memory"

Reply:
@dipguptaredhat don't forget to resolve the conversation once is done from your side! ;)